environs/context/callcontext.go

```diff
@@ -7,51 +7,67 @@ import (
 	stdcontext "context"
```

Comment:
```
package context
```

I think it might be time to rename this package to `envcontext`, to stop overloading the standard `context` package name.

Reply:
Lots more files changed now